source/wind.c

```diff
@@ -323,46 +333,69 @@ model_vgrad (ndom, x, v_grad)
      int ndom;
 {
 
-  double v0[3], v1[3];
-  double dx[3], dv[3];
+  double v[3], v_forward[3], v_reverse[3];
+  double dx_forward[3], dx_reverse[3];
+  double dv[3];
   double ds;
   int i, j;
-  int vsub (), stuff_v ();
+  double zero_vector[3], dx[3], dx_obs[3];
+
+  zero_vector[0] = zero_vector[1] = zero_vector[2] = 0.0;
+
+
+
+
+  ds = 0.00001 * length (x);
```

Comment:
I think this ds is in the cmf frame, right? Should we label it as ds_cmf instead?

Reply:
Agreed.  I will make this change.